packages/now-routing-utils/test/superstatic.spec.js

```diff
@@ -301,6 +301,15 @@ test('convertRewrites', () => {
     { source: '/some/old/path', destination: '/some/new/path' },
     { source: '/firebase/(.*)', destination: 'https://www.firebase.com' },
     { source: '/projects/:id/edit', destination: '/projects.html' },
+    {
+      source: '/users/:id',
+      destination: '/api/user?identifier=:id&version=v2',
+    },
```

Comment:
should we also test a destination that has a segment in the `path` and `query` e.g. `/api/:user?identifier=:id`?

Reply:
@ijjk Great idea! Done ✅

Reply:
Oh good catch with the hash one too 👍